prediction/src/tests/test_endpoints.py

```diff
@@ -4,13 +4,13 @@
 
     Provides unit tests for the API endpoints.
 """
+from flask import url_for
 from functools import partial
 import json
-
 import pytest
-
-from flask import url_for
-from src.factory import create_app
+import sys
+sys.path.append('.')
+from src.factory import create_app  # NOQA
```

Comment:
Removing `interface/__init__.py` and `prediction/__init__.py` makes the tests still pass but Sphinx unable to detect interface and prediction as package so it won't be able to automatically summarize the classes using their docstrings.